canvas_sdk/commands/tests/test_utils.py

```diff
@@ -22,6 +22,17 @@
 from canvas_sdk.commands.constants import Coding
 
 
+class Secret:
+    def __init__(self, value):
```

Comment:
I think that this should be named `Token`. `Secret` might be confused with either the client secret from the API credentials or the secrets set for plugins.

Reply:
I went with `MaskedValue`.